libmscore/tempotext.h

```diff
@@ -46,11 +47,13 @@ class TempoText : public Text  {
 
       qreal tempo() const        { return _tempo;      }
       void setTempo(qreal v);
+      bool isRelative() { return _relative!=0; }
```

Comment:
Or compare to (close to) 1.0, as I'm not sure whether 1.0 has an exact represenation of a float. If it has, mentione in a comment